### PR TITLE
Updates to the cosmwasm documentation

### PR DESCRIPTION
**Pull request by emperorjm**
- Addition of a new Getting Started section
- Updates to the Architecture section
- Updates to the IDE Setup guide

### CODE REVIEW

File: content/2.developers/4.cosmwasm-documentation/2.getting-started/1.introduction.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+objectID: developers|cosm_wasm|getting-started|introduction
+title: Introduction
+description: Getting started with CosmWasm development
+parentSection: CosmWasm
+parentSectionPath: /developers/cosmwasm-documentation/introduction
+---
+
+# Getting started
+
+The getting started section offers a tour of the fundamental aspects of CosmWasm Smart Contracts. Although a basic understanding of Rust would be helpful, the content is intended for learners of all levels, regardless of their experience. The aim is to provide easy-to-follow instructions and give first-time users hands-on experience by walking them through a step-by-step guide that covers the following topics:
```
Comment (lufa23) [inline content/2.developers/4.cosmwasm-documentation/2.getting-started/1.introduction.md:11]: "first-time users **a** hands-on experience"

File: content/2.developers/4.cosmwasm-documentation/2.getting-started/6.interact-with-contract.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Deploy contract
+
+In the [Compile Contract](/developers/cosmwasm-documentation/getting-started/compile-contract) article, we generated a wasm binary executable. You can upload the code to the blockchain, and once the process is complete, you can download the bytecode for verification purposes.
+
+### Deploy using archwayd
+
+Now you will store the wasm bytecode of the `cw_namespace` contract on chain and obtain the `code id`. This `code id` will be used later to create an instance of the `cw_namespace` contract.
+
+`mywallet` should be the name of the your wallet created in the [Setting up environment](/developers/cosmwasm-documentation/getting-started/setting-up-environment) article. Execute the following command to store the contract on chain:
```
Comment (lufa23) [inline content/2.developers/4.cosmwasm-documentation/2.getting-started/6.interact-with-contract.md]: `mywallet` should be the name of the wallet

File: content/2.developers/4.cosmwasm-documentation/3.architecture/1.multi-chain-contracts.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+parentSection: Architecture
+---
+
+# What are multi-chain contracts?
+CosmWasm is designed and built from the ground up to be a multi-chain solution for smart contracts. As it comes from the Cosmos ecosystem, it is designed for networks of blockchains, rather than siloed chains. But what exactly do we mean by multi-chain?
+
+## Different chain, same contract
+All code is designed to be agnostic to the details of the underlying chain, so by writing a CosmWasm contract, you will be able to run it on different chains within the Cosmos ecosystem.
+
+## Inter blockchain contracts
```
Comment (lufa23) [inline content/2.developers/4.cosmwasm-documentation/3.architecture/1.multi-chain-contracts.md:14]:
```suggestion
## Inter-blockchain contracts
```

**Review by lufa23 — COMMENTED** [review]
just few mispells

File: content/2.developers/4.cosmwasm-documentation/3.architecture/1.multi-chain-contracts.md (hunk 3 above)
Comment (lufa23) [reply content/2.developers/4.cosmwasm-documentation/3.architecture/1.multi-chain-contracts.md:14]: i am wondering, should we call them Multi-Chain contracts directly?

**Review by lufa23 — DISMISSED** [review]

File: content/2.developers/4.cosmwasm-documentation/2.getting-started/6.interact-with-contract.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+# The two binaries should be identical
+diff artifacts/cw_nameservice.wasm download.wasm
+```
+
+If the `diff` command returns a blank output, it means that the two files being compared are identical.
+
+### Deploy using archway developer cli
+
+If your contract was created using the `archway new` command, you can deply the contract using the following command:
```
Comment (emperorjm) [inline content/2.developers/4.cosmwasm-documentation/2.getting-started/6.interact-with-contract.md]:
```suggestion
If your contract was created using the `archway new` command, you can deploy the contract using the following command:
```

**Review by lufa23 — APPROVED** [review]

**Pull request merged**